lib/features/sks-menu/data/api_client/sks_menu_client.dart

```diff
@@ -0,0 +1,19 @@
+import "package:dio/dio.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:riverpod_annotation/riverpod_annotation.dart";
+
+import "../../../../config/env.dart";
+
+abstract class SksMenuClientConfig {
+  static final rootUrl = Env.sksUrl;
+  static const mealsEndpoint = "/meals";
+}
+
+@riverpod
+Dio sksClient(Ref ref) {
```

Comment:
I think in Tomek PR we had the same client so this can be merged together into one client (the base url is the same)